app/views/users/edit.html.erb

```diff
@@ -106,3 +72,37 @@
     <% end %>
   </div>
 </div>
+
+
+<%= content_for :javascript do %>
+<script type="text/javascript">
+$(".hoverable").hover(function () {
+  $(this).toggleClass("hover");
+});
+
+$('#enable-other-sign-in').click(function(e) {
+  e.preventDefault();
+  $(this).hide();
+  $('.row.other-sign-in').slideToggle();
+});
+
+$.fn.editable.defaults.mode = 'inline';
+$.fn.editable.defaults.send = "always";
+$.fn.editable.defaults.ajaxOptions = {type: "PUT"};
+$.fn.editable.defaults.url = "<%= profile_path %>";
+
+$('#username').editable();
+
+$('#name').editable({
+  value: {
+    title: "<%= current_user.title %>",
+    first_name: "<%= current_user.first_name %>",
+    last_name: "<%= current_user.last_name %>",
+    suffix: "<%= current_user.suffix %>"
+  },
+  success: function(response, newValue) {
+    $(this).html(response.full_name);
+  }
+});
+</script>
```

Comment:
Yes, and I started to do so, but then ran into the issue that they use `erb` interpolation that's not available on an asset, only a view.  Well I suppose an asset can use it, but it doesn't have access to the current user, since the `ERB` is only ran once at compilation :grin: